### PR TITLE
Deal with partial fixity (only x / only y)

### PR DESCRIPTION
**Pull request by ricardoavelino**
This pull request includes the attributes ``fix_x``and ``fix_y`` to the ``__init__()`` of ``compas_ags.diagrams.formdiagram``. 

With these attributes, constraints can be set up based on them.

I also added one example file and `.json` with the geometry of the example.

### CODE REVIEW

**Comment by ricardoavelino** [comment]
Please @brgcode review this PR before #106 ;)

File: src/compas_ags/diagrams/formdiagram.py [hunk 1]
```diff
@@ -22,6 +22,8 @@ def __init__(self):
         })
         self.update_default_vertex_attributes({
             'is_fixed': False,
+            'fix_x': False,
+            'fix_y': False,
```
Comment (brgcode) [inline src/compas_ags/diagrams/formdiagram.py:26]: we tend to use the convention that attributes with boolean values have an `is_` prefix...

Comment (ricardoavelino) [reply src/compas_ags/diagrams/formdiagram.py]: changed

Comment (ricardoavelino) [reply src/compas_ags/diagrams/formdiagram.py]: now they are 'is_fixed_x' and 'is_fixed_y'

**Pull request merged**